### PR TITLE
Get attachment type fix

### PR DESCRIPTION
**Pull request by shilwantk**
## Summary
Added a return statement for cases when no contentPrefix is present in the fileMetaData

### CODE REVIEW

File: Sources/Utilities/ALMessage+Extension.swift [hunk 1]
```diff
@@ -345,13 +345,12 @@ extension ALMessage {
     }
 
     private func getAttachmentType() -> ALKMessageType? {
-        guard let fileMeta = fileMeta else { return nil }
         
-        var contentPrefix = String()
-        if let index = fileMeta.contentType.firstIndex(of: "/") {
-            contentPrefix = String(fileMeta.contentType.prefix(upTo: index))
-        }
+        guard let fileMeta = fileMeta, (fileMeta.contentType != nil) else { return nil }
```
Comment (mukeshthawani) [inline Sources/Utilities/ALMessage+Extension.swift]: You can create a variable here and use it below: `let contentType = fileMeta.contentType`.

Comment (shilwantk) [reply Sources/Utilities/ALMessage+Extension.swift]: Have added the change.

Comment (mukeshthawani) [reply Sources/Utilities/ALMessage+Extension.swift]: Replace this line: `(fileMeta.contentType != nil)` with: `let contentType = fileMeta.contentType`.

Comment (shilwantk) [reply Sources/Utilities/ALMessage+Extension.swift]: Changes done.

**Review by mukeshthawani — APPROVED** [review]

**Pull request merged**